config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
- root 'homepage#index'
+  resources :clients
```

Comment:
give some line breaks in the file to make things less cluttered

Reply:
line breaks added.